### PR TITLE
Home Location Corrections

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#19506

*Description of changes:*
* Connect MHR baseline and current states to Updated Badges for Home Location components in MHR Correction Flow
* Unique messaging for moving into/out of province in Civic Address Region Corrections

Screenshot 2024-03-20 at 2 52 01 PM
Screenshot 2024-03-20 at 2 49 49 PM
Screenshot 2024-03-20 at 2 49 45 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1791-upn1qbqn.web.app

File: ppr-ui/src/views/newMhrRegistration/HomeLocation.vue [hunk 1]
```diff
@@ -135,6 +140,24 @@ export default defineComponent({
...
+      homeLocationCorrection: computed((): UpdatedBadgeIF => {
+        return {
+          baseline: { ...getMhrBaseline.value?.location, address: null },
+          currentState: { ...getMhrRegistrationLocation.value, address: null }
+        }
+      }),
+      civicAddressCorrection: computed((): UpdatedBadgeIF => {
+        return {
+          baseline: getMhrBaseline.value?.location.address,
+          currentState: getMhrRegistrationLocation.value.address
```
Comment (dimak1) [inline ppr-ui/src/views/newMhrRegistration/HomeLocation.vue:153]: You might want to use computed for the `currentState`. In my case, when working with Views/Components that are not expected to have badges, I got errors for the `currentState` props being undefined.

**Review by dimak1 — APPROVED** [review]
Looks good, just one comment to check.

**Review by doug-lovett — APPROVED** [review]
Very nice.

File: ppr-ui/src/views/newMhrRegistration/HomeLocation.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/views/newMhrRegistration/HomeLocation.vue:153]: I will gave it a double check, and see no console errors in MHR, Mhr Information view (ie transport permits/amends) or Mhr Corrections. 
The object itself is a computed value as both properties are derived from another source, that are subject to change.

Comment (dimak1) [reply ppr-ui/src/views/newMhrRegistration/HomeLocation.vue:153]: Oh right sorry, I missed the object itself is computed haha, nevermind!

Comment (cameron-eyds) [reply ppr-ui/src/views/newMhrRegistration/HomeLocation.vue:153]: All good!

**Pull request merged**